coalib/bearlib/languages/definitions/Haskell.py

```diff
@@ -0,0 +1,16 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class Haskell:
+    extensions = '.hs',
+    versions = 1.0, 1.1, 1.2, 1.3, 1.4
+    special_chars = list('!#$%&*+./<=>?@\\^|-~')
+    comment_delimiters = '--',
+    multiline_comment_delimiters = {'{-': '-}'}
+    keywords = [
+        'case', 'class', 'data', 'default', 'deriving', 'do', 'else',
+        'forall', 'if', 'import', 'in', 'infix', 'infixl', 'infixr',
+        'let', 'module', 'newtype', 'of', 'qualified', 'then', 'type',
+        'where', '_', 'foreign', 'ccall', 'as', 'safe', 'instance', 'unsafe']
+    type_declaration = {'::'}
```

Comment:
Sorry if https://github.com/coala/coala/pull/5759#discussion_r217915557 is confusing. What I mean is this field should better be removed.